metabase/metabase
# Migrate UserProfileForm to formik

Epic https://github.com/metabase/metabase/issues/26178

How to test:
- Account -> Profile
- Make sure it is possible to edit and save all the fields

<img width="586" alt="Screenshot 2022-11-03 at 17 21 48" src="https://user-images.githubusercontent.com/8542534/199762209-447e2e45-42f2-4bce-b2ef-1569e2fa67cc.png">
